src/dataflow/src/logging/materialized.rs

```diff
@@ -83,14 +84,15 @@ pub struct Peek {
     id: GlobalId,
     /// The logical timestamp requested.
     time: Timestamp,
-    /// The connection ID of the peek.
-    conn_id: u32,
+    /// The ID of the peek.
```

Comment:
I'm not really sure how the logging framework is used. Would it be worth it to try and include the connection ID in all Peek logging events?

Reply:
Eh, nah, I'd skip it. We can just add a log line to the moment that a peek ID is minted, if we need the association between conn ID -> peek ID to be visible in the logs. (Or, something something, distributed tracing spans, something something. Point is: don't add data here that's not necessary for the sake of logging. There are otherwise to recover that information in logging if it becomes necessary.)

Reply:
Great question. I'm not sure about the answer but imo its ok. The coordinator has this information, and we could log it there and join later (no need to muck up abstraction boundaries just to stash some "log me" data).